xml/System.Collections.Generic/PriorityQueue`2.xml

```diff
@@ -26,7 +26,7 @@
 
 ## Remarks
 
-Implements an array-backed, quaternary min-heap. Each element is enqueued with an associated priority that determines the dequeue order. Elements with the lowest priority are dequeued first.
+Implements an array-backed, quaternary min-heap. Each element is enqueued with an associated priority that determines the dequeue order. Elements with the lowest priority are dequeued first. Note that the type does not guarantee first-in-first-out semantics for elements of equal priority.
```

Comment:
```suggestion
Implements an array-backed, quaternary min-heap. Each element is enqueued with an associated priority that determines the dequeue order. Elements with the lowest priority are dequeued first. This type does not guarantee first-in-first-out semantics for elements of equal priority.
```

Reply:
My PR got auto-merged, so your suggestion ended up getting ignored :-(

Reply:
Yes, I wish GitHub didn't automerge if there are suggestions. I don't think it's important enough to create a new PR though.